src/Sylius/Bundle/ApiBundle/Validator/Constraints/EligibleProductVariantToCartValidator.php

```diff
@@ -14,59 +14,74 @@
 namespace Sylius\Bundle\ApiBundle\Validator\Constraints;
 
 use Sylius\Bundle\ApiBundle\Command\Cart\AddItemToCart;
+use Sylius\Bundle\ApiBundle\Command\Cart\ChangeItemQuantityInCart;
 use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Model\OrderItemInterface;
 use Sylius\Component\Core\Model\ProductInterface;
 use Sylius\Component\Core\Model\ProductVariantInterface;
 use Sylius\Component\Core\Repository\OrderRepositoryInterface;
 use Sylius\Component\Inventory\Checker\AvailabilityCheckerInterface;
 use Sylius\Component\Product\Repository\ProductVariantRepositoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
 use Symfony\Component\Validator\Constraint;
 use Symfony\Component\Validator\ConstraintValidator;
 use Webmozart\Assert\Assert;
 
 /** @experimental */
-final class AddingEligibleProductVariantToCartValidator extends ConstraintValidator
+final class EligibleProductVariantToCartValidator extends ConstraintValidator
```

Comment:
```suggestion
final class EligibleProductVariantInCartValidator extends ConstraintValidator
```

You should mention this change in UPGRADE file as this service has been added in `1.9`

Reply:
This service was added few days ago
<img width="701" alt="Screenshot 2021-05-20 at 12 35 46" src="https://user-images.githubusercontent.com/39232096/118964642-fe0d8100-b967-11eb-81cd-2812646e5e27.png">

Reply:
It is there: https://github.com/Sylius/Sylius/blob/1.9/src/Sylius/Bundle/ApiBundle/Validator/Constraints/AddingEligibleProductVariantToCartValidator.php, but not tagged yet https://github.com/Sylius/Sylius/blob/v1.9.3/src/Sylius/Bundle/ApiBundle/Validator/Constraints/AddingEligibleProductVariantToCartValidator.php